core/cap-0033.md

```diff
@@ -68,8 +68,14 @@ typedef AccountID* SponsorshipDescriptor;
 
 struct AccountEntryExtensionV2
 {
-    uint32 numSponsored;
-    uint32 numSponsoring;
+    uint32 numSponsored;  // Number of reserves sponsored by this account
+                          // This reduces the base reserve requirement
+    uint32 numSponsoring; // Number of reserves sponsored for this account
+                          // This increases the base reserve requirement
```

Comment:
I'm still very confused by these descriptions. I thought I understand until I read the second comment on each. Is `numSponsored` reducing the base reserve of this account or another account?

Reply:
Clarified, let me know if you think it can use more work. I really want to be precise about this.